File: pset1/mario/more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    int s;
    int x;
    int j;
    // get user input
    do
    {
        n = get_int("Pyramid height: ");
    }
    while (n < 0 || n > 23);

    // Bricks
    for (x = 0; x < n; x++)
    {
        // Spaces
        for (s = 0; s < n; s++)
        {
            if (s + x < n - 1)
            {
                // spaces
                printf(" ");
            }
            else
            {
                // bricks
                printf("#");
            }
        }

        // separation between half pyramids
        printf("  ");

        // second half of pyramid
        for (j = 0; j < x + 1; j++)
        {
            if (j <= x)
            {
                printf("#");
            }
        }
        printf("\n");
    }

}

File: pset6/mario/more/mario.py
import cs50

while True:
    # get size of pyramid from the user
    n = cs50.get_int("Number please between 0 - 23: ")
    if n >= 0 and n <= 23:
        break

for x in range(n):
    # outerLoop
    for s in range(n):
        # innerLoop
        if s + x < n - 1:
            # spaces
            print(" ", end="")
        else:
            # bricks
            print("#", end="")

    # separation
    print("  ", end="")

    # second half of pyramid
    for j in range(x + 1):
        if j <= x:
            print("#", end="")
    print()
